content/en/docs/gke/deploy/management-setup.md

```diff
@@ -6,22 +6,26 @@ weight = 3
 
```

Comment:
Line 3:

- Are we still sticking to using GCP _and_ Google Cloud?

Also - strange brackets:

```diff
- description = "Instructions for setting up a management cluster on (GCP)"
+ description = "Instructions for setting up a management cluster on GCP"
```
or
```diff
+ description = "Instructions for setting up a management cluster on Google Cloud"

Reply:
Yeah, they were written before we stick to the pattern. Thanks for pointing out!